src/rez/cli/test.py

```diff
@@ -67,3 +95,75 @@ def command(opts, parser, extra_arg_groups=None):
 
         if returncode:
             sys.exit(returncode)
+
+
+def is_dev_run(name):
+    """
+    Decides if is a dev_run, based on the package name
+    Args:
+        name: name of package
+
+    Returns: True or False
+    """
+    return not name or name == '.'
+
+
+def get_package(path):
+    """
+    Function validates pkg name and indicates if tests were run in developer mode
+    Args:
+        path: path to the package
+
+    Returns: Tuple of values (package_name, is_dev_run)
+    """
+    try:
+        return get_developer_package(path)
+    except PackageMetadataError:
+        print("Not a valid rez package. Make sure you are in package's root directory", file=sys.stderr)
+        return None
+
+
+def prepare_dev_env_package(package):
```

Comment:
I'm actually pretty hesitant about rez-test possibly implicitly installing a package like this. Package installation is always explicit everywhere else. Furthermore this feels arbitrary - the package gets installed if it wasn't there, but what if it's there but put of date due to a sourcecode change? We can't detect that and I don't think we should.

It seems simpler and less error prone to simply give an error if the current developer package isn't present as a local package, and leave it up to the user if and when to rebuild/reinstall the package.

Reply:
yes, we can not detect that. ATM we are only detecting changes on the packages.py and reinstalling the package when the tests are out of date. 
I'll do a poll tomorrow when I'm at work, I'm guessing not many devs are really relying on this feature. So, I guess it can go away